components/welcome-mat/__examples__/trailhead-complete.jsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import WelcomeMat from '~/components/welcome-mat';
+import WelcomeMatTile from '~/components/welcome-mat/tile';
+import WelcomeMatInfoBadge from '~/components/welcome-mat/info-badge';
+import Button from '~/components/button';
+import IconSettings from '~/components/icon-settings';
+
+class Example extends React.Component {
+	static displayName = 'welcomeMatExample';
+
+	render() {
+		return (
+			<IconSettings iconPath="/assets/icons">
+				<div style={{ position: 'relative', height: '5rem' }}>
+					<div style={{ width: '20rem' }}>
+						<WelcomeMat
+							id="welcome-mat-trailhead-complete-example"
+							labels={{
+								title: 'The Lightning Experience is here!',
+								description:
+									'Welcome to Lightning Experience, the modern, beautiful user experience from Salesforce. With a sales-and service-centric mindset, we focused on reinventing the desktop environment to better support your business processes."',
+							}}
+							infoBadge={() => (
```

Comment:
Update this to pass the WelcomeMatInfoBadge here directly rather than in a function